dell/xps/15-9500/nvidia/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, pkgs, ... }:
+
+# This creates a new 'nvidia-offload' program that runs the application passed to it on the GPU
+let
+  nvidia-offload = pkgs.writeShellScriptBin "nvidia-offload" ''
```

Comment:
Can you put a simple example in the README like

To run using the nvidia gpu use this:

```
$ nvidia-offload nix-shell -p glxinfo --run 'glxgears'
```